src/Dfc.CourseDirectory.Core/Search/SearchResult.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace Dfc.CourseDirectory.Core.Search
+{
+    public class SearchResult<TResult>
+        where TResult : class
+    {
+        public IEnumerable<TResult> Results { get; set; }
+
+        public IDictionary<string, IDictionary<object, long?>> Facets { get; set; }
+
+        public long? TotalCount { get; set; }
```

Comment:
Why nullable?

Reply:
That's what is returned from the `Azure.Search.Documents.Models.FacetResult` object.